assets/schema_input.json

```diff
@@ -41,18 +41,18 @@
                 "meta": ["lane"]
             },
             "fastq_1": {
-                "errorMessage": "FastQ file for reads 1 cannot contain spaces, has to be a string, has to exist and must have extension '.fq.gz' or '.fastq.gz'",
                 "type": "string",
-                "pattern": "^\\S+\\.f(ast)?q\\.gz$",
                 "format": "file-path",
-                "exists": true
+                "exists": true,
+                "pattern": "^([\\S\\s]*\\/)?[^\\s\\/]+\\.f(ast)?q\\.gz$",
```

Comment:
What is the change here?

Reply:
TEMPLATE, no idea what this regex of hell does